rgw/v2/tests/s3_swift/test_multisite_bucket_granular_sync_policy.py

```diff
@@ -270,9 +278,92 @@ def test_exec(config, ssh_con):
+                                        f"object should be sync to another site for bucket {bkt.name}, but not synced"
                                     )
 
+                        if config.test_ops.get("write_io_verify_another_site", False):
+                            _, stdout, _ = rgw_ssh_con.exec_command(
+                                f"radosgw-admin bucket stats --bucket {bkt.name}"
+                            )
+                            cmd_output = json.loads(stdout.read().decode())
+                            num_objects = (
+                                cmd_output["usage"]["rgw.main"]["num_objects"]
+                                if "rgw.main" in cmd_output["usage"].keys()
+                                else 0
+                            )
+
+                            other_site_bucket = s3lib.resource_op(
+                                {
+                                    "obj": other_site_rgw_conn,
+                                    "resource": "Bucket",
+                                    "args": [bkt.name],
+                                }
+                            )
+                            for oc, size in list(config.mapped_sizes.items()):
+                                config.obj_size = size
+                                s3_object_name = "new-" + utils.gen_s3_object_name(
+                                    bkt.name, oc
+                                )
+                                log.info(f"s3 object name: {s3_object_name}")
+                                s3_object_path = os.path.join(
+                                    TEST_DATA_PATH, s3_object_name
+                                )
+                                log.info(f"s3 object path: {s3_object_path}")
+                                log.info("upload type: normal")
+                                reusable.upload_object(
+                                    s3_object_name,
+                                    other_site_bucket,
+                                    TEST_DATA_PATH,
+                                    config,
+                                    each_user,
+                                )
+
+                            _, stats_stdout, _ = rgw_ssh_con.exec_command(
+                                f"radosgw-admin bucket stats --bucket {bkt.name}"
+                            )
+                            re_cmd_output = json.loads(stats_stdout.read().decode())
+                            log.info(f"re_cmd_output : {re_cmd_output}")
+                            new_object_count = num_objects + config.objects_count
+                            if (
+                                re_cmd_output["usage"]["rgw.main"]["num_objects"]
+                                != new_object_count
+                            ):
+                                raise TestExecError(
+                                    f"Failed to upload object to bucket {bkt.name}"
```

Comment:
@anrao19 could you modify the exception message to "number of objects not updated in the bucket stats after new objects upload"

Reply:
@hmaheswa changes made